sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java

```diff
@@ -254,6 +317,68 @@ public boolean accept(File dir, String filename) {
         return files != null ? files : new File[0];
     }
 
+    /**
+     * Look for 'deviceinfo' file inside the minidump folder and parse it.
+     *
+     * @param logFolder - folder where to look for stored device information.
+     * @return a device information or null.
+     */
+    @Nullable
+    public static Device getStoredDeviceInfo(File logFolder) {
+        File[] files = logFolder.listFiles(new FilenameFilter() {
+            @Override
+            public boolean accept(File dir, String filename) {
+                return filename.equals(DEVICE_INFO_FILE);
+            }
+        });
+        if (files == null || files.length == 0) {
+            AppCenterLog.error(Crashes.LOG_TAG, "No stored deviceinfo file in a minidump folder.");
```

Comment:
```suggestion
            AppCenterLog.warn(Crashes.LOG_TAG, "No stored deviceinfo file in a minidump folder.");
```

And we should fall back to current device object like before in crashes.